lib/generators/react_on_rails/base_generator.rb

```diff
@@ -6,7 +6,7 @@
 module ReactOnRails
   module Generators
     class BaseGenerator < Rails::Generators::Base # rubocop:disable Metrics/ClassLength
-      hide!
+      Rails::Generators.hide_namespace self.namespace
```

Comment:
Is this what you are looking for?
http://apidock.com/rails/Rails/Generators/hide_namespace/class